observability/otel/otel-rn.adoc

```diff
@@ -1,7 +1,7 @@
 :_mod-docs-content-type: ASSEMBLY
 include::_attributes/common-attributes.adoc[]
 [id="otel_rn"]
-= Release notes for the {OTELName}
+= Release notes of the {OTELName}
```

Comment:
A search in Atom shows 0 instances of _Release notes of_ and 155 _Release notes for_.  Is there a specific reason for this change?

Reply:
Not really. I only wanted to shorten it.
But thank you for the feedback.
I will undo this change and leave it as "for".